core/src/main/java/com/dtolabs/rundeck/plugins/scm/JobImporter.java

```diff
@@ -62,10 +62,11 @@
      * @param input          input stream
      * @param importMetadata metadata to attach to the job
      * @param preserveUuid   if true, preserve any UUID on import, otherwise remove it
+     * @param renamedJob     pass original ID/UUID if the job is renamed
      *
      * @return result
      */
-    ImportResult importFromStream(String format, InputStream input, Map importMetadata, boolean preserveUuid);
+    ImportResult importFromStream(String format, InputStream input, Map importMetadata, boolean preserveUuid, Map renamedJob);
```

Comment:
looking at where this `renamedJob` is used, it would be better if it was not a Map, eg. add a new interface that has `getUuid()` and `getSourceId()`